### ISSUE TITLE
Feature request: support external CSS file for configuration

### ISSUE THREAD

**Issue by magicDGS** [caksoylar/keymap-drawer#135]
One of the major problems of using custom CSS with the tool is that it is written in-line on the YAML file, with doesn't allow to format, lint, or syntax-hightlight on editors. This makes maintenance of custom CSS configuration (either extra configuration or direct override) more difficult.

It would be nice if the `svg_style` and `svg_style_dark` and `svg_extra_style` support to provide a relative file-path to use instead of being used directly from the yaml.

Last but not least, a dump command for the default CSS file would be very nice for those that wanna modify the whole style and not only add extra, or if they want to explore what is defined to be able to override with the proper specificity.

**Comment by caksoylar** [comment]
I agree, I'll look into solving the relative path issue that is also needed for #85. I don't think it is useful if the path isn't interpreted relative to the config file path.

I am not sure about dumping the CSS: `keymap dump-config` originally dumped all fields but I excluded some fields (including `svg_style`) later because I'd prefer people use the `svg_extra_style`. A lot of things will break if the base style is not updated along with the code, because everyone is overwriting it in their config.

I think being able to easily inspect it is a fair point though. What classes exist etc. is not very discoverable right now unless you look at the CSS, like you noted. Maybe when there is support for file includes, I'll move them to separate files in the repo and link to the files directly in the docs and the CLI help.
